### ISSUE TITLE
clang warning in ppport.h

### ISSUE THREAD

**Issue by pmqs** [pmqs/Compress-Raw-Zlib#1]
```
In file included from Zlib.xs:85:
./ppport.h:4385:10: warning: 'WIDEST_UTYPE' macro redefined [-Wmacro-redefined]
# define WIDEST_UTYPE U64TYPE
         ^
/home/paul/base/install/clang-8/perl/std/5.30.0/lib/5.30.0/x86_64-linux/CORE/handy.h:1072:12: note: 
      previous definition is here
#   define WIDEST_UTYPE U64
           ^
1 warning generated.
```

```
$ clang --version
clang version 8.0.0-3 (tags/RELEASE_800/final)
Target: x86_64-pc-linux-gnu
Thread model: posix
InstalledDir: /usr/bin
```

**Comment by pmqs** [comment]
Thanks - looks like updating to latest ppport.h silences the issue.

**Comment by pmqs** [comment]
Fixed with 664a5fbacf778acdd4cfbcc571997f3df5ee43d3
